### PR TITLE
Rename kubernetes tentacle to agent

### PR DESCRIPTION
**Pull request by scme0**
We have decided to rename the tentacle to an agent as the tentacle component is only part of the complete agent.

### CODE REVIEW

File: charts/kubernetes-agent/.changeset/spicy-paws-float.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"kubernetes-agent": patch
```
Comment (APErebus) [inline charts/kubernetes-agent/.changeset/spicy-paws-float.md:2]: let's go minor here, so we get a meaningful-ish version bump

File: charts/kubernetes-agent/values.yaml [hunk 2]
```diff
@@ -1,11 +1,11 @@
-# Default values for kubernetes-tentacle.
+# Default values for kubernetes-agent.
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
 replicaCount: 1
 
 image:
-  repository: docker.packages.octopushq.com/octopusdeploy/kubernetes-tentacle
+  repository: docker.packages.octopushq.com/octopusdeploy/kubernetes-agent
```
Comment (APErebus) [inline charts/kubernetes-agent/values.yaml:8]: this won't change as the tentacle image will still be `tentacle`

**Review by APErebus — CHANGES_REQUESTED** [review]

File: charts/kubernetes-agent/values.yaml (hunk 2 above)
Comment (scme0) [reply charts/kubernetes-agent/values.yaml]: Damn, good catch!

**Review by APErebus — APPROVED** [review]

**Comment by scme0** [comment]
[sc-69136]

**Pull request merged**